apps/web/src/pages/UserEdit/UserEdit.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useForm, FormProvider } from 'react-hook-form';
+import { useGetCurrentUserQuery } from 'apollo-hooks';
+
+import { Container } from './style';
+import { NextSeo } from 'next-seo';
+
+import EditProfileFormModal from './EditProfileForm';
+
+const UserEdit = () => {
+  const { data } = useGetCurrentUserQuery();
+
+  const methods = useForm();
+
+  return (
+    <Container>
+      <FormProvider {...methods}>
+        <EditProfileFormModal userDetails={data?.getCurrentUser} />
```

Comment:
Please bring all the contents of `EditProfileForm` here, no need to have it on a separate component, there is nothing else here.

Reply:
done